src/deployments.ts

```diff
@@ -42,62 +44,71 @@ export const DEPLOYMENT_SYLVESTER_POLYGON_MAINNET_V0 = {
   network: NETWORK_POLYGON_MAINNET,
   contractType: RenftContractType.SYLVESTER,
   version: SylvesterVersion.V0,
+  startBlock: 28399140,
 } as const;
 
 export const DEPLOYMENT_SYLVESTER_POLYGON_MAINNET_V1 = {
   contractAddress: '0x4e52b73aa28b7ff84d88ea3a90c0668f46043450',
   network: NETWORK_POLYGON_MAINNET,
   contractType: RenftContractType.SYLVESTER,
   version: SylvesterVersion.V1,
+  startBlock: 36825974,
 } as const;
 
 export const DEPLOYMENT_WHOOPI_AVALANCHE_FUJI_TESTNET_V0 = {
```

Comment:
This is strange, this address looks to be an ERC20: https://testnet.snowtrace.io/address/0x42816FA3cB0aDc3fcAdED3109323c0Bc19215084#code

Can we fix this regression?

Reply:
That is a polygon mainnet deployment, not AVAX